### PR TITLE
Suggested tweaks

### PR DESCRIPTION
**Pull request by estelle**

### CODE REVIEW

File: src/index.njk [hunk 1]
```diff
@@ -14,7 +14,10 @@ title: The next step for member invitations
   entirely unchecked. While we’re excited to be part of a larger extended
   ‘fediverse’ community, we need to keep this server at a size that we’re
   able to afford, maintain, and moderate. We think that number is somewhere
-  under 500 members. It’s impossible to draw a line around a community, but we
+  under 350 members.</p>
```
Comment (estelle) [inline src/index.njk:17]: we should start off smaller and leave room to grow

File: src/index.njk [hunk 2]
```diff
@@ -5,7 +5,7 @@ title: The next step for member invitations
 <p>Hey community!</p>
 
 <p>We're so grateful for the enthusiasm we've received in these first days of
-  front-end.social and we're enjoying getting to know new faces (as well as
+  <a href="/">front-end.social</a> and we're enjoying getting to know new faces (as well as
```
Comment (estelle) [inline src/index.njk:8]: subdomain, so we should link to the main page.

Comment (estelle) [inline src/index.njk:8]:
```suggestion
  <a href="https://front-end.social">front-end.social</a> and we're enjoying getting to know new faces (as well as
```

**Review by mirisuzanne — APPROVED** [review]
looks good to me!

**Pull request merged**